bulbs/feeds/views.py

```diff
@@ -56,3 +64,20 @@ def get_queryset(self):
             return self.model.objects.none()
 
         return sc.get_content()[:self.paginate_by]
+
+
+class GlanceFeedViewSet(viewsets.ReadOnlyModelViewSet):
+
+    model = Content
+    serializer_class = GlanceContentSerializer
+
+    queryset = Content.search_objects.search().sort('-last_modified').filter(Published())
+
+    permission_classes = [AllowAny]
```

Comment:
`permission_classes` is usually a tuple, so this would be `permission_classes=(AllowAny,)`